build/SignList.xml

```diff
@@ -0,0 +1,20 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemGroup>
+    <FirstParty Include="Xamarin.Forms.*.dll" />
+    <FirstParty Include="FormsViewGroup.dll" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ThirdParty Include="GMap.NET.Core.dll" />
+    <ThirdParty Include="GMap.NET.GTK.dll" />
+    <ThirdParty Include="webkit-sharp.dll" />
+  </ItemGroup>
```

Comment:
Mono.Cecil needs to be added here. 

https://github.com/xamarin/AndroidX/pull/193

```suggestion
    <ThirdParty Include="Mono.Cecil.dll" />
    <ThirdParty Include="Mono.Cecil.*.dll" />
  </ItemGroup>
```

Reply:
Added!